### ISSUE TITLE
Hello - I'm relatively new to the C. elegans resea...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#7025]
*Help Desk query collected when no chat operators were online. Follow up required.*

Hello - I'm relatively new to the C. elegans research community. What does it mean if a gene is listed as "This gene is dead!"/(Status: Dead)/etc. ? (e.g. WBGene00091073)

Thanks!

**Reported by:** Mich********** (mrie************)
**Submitted from:** <a target="_blank" href="http://www.wormbase.org/https://www.wormbase.org/species/p_pacificus/gene/WBGene00091073#0-9g-3">https://www.wormbase.org/species/p_pacificus/gene/WBGene00091073#0-9g-3</a>
**Browser:** Chrome 72.0.3626.121

**Comment by vanaukenk** [comment]
Dear Michael,


Thanks for writing to WormBase.


"Dead" refers to the status of the gene entry in the database. 
Live genes can be updated to "dead" status for various reasons, but we retain the original gene id in WormBase and try to provide some indication in a textual remark as to why the status changed.
For WBGene00091073, that remark is " [241117 mh6] removed as it does not map to the new gene set".

Our sequence curators at the EBI in Hinxton may be able to provide more specific details about this remark, but in general "dead" genes are no longer part of the valid gene set of the particular genome in question.

Please let us know if you have any additional questions.

Best,

--Kimberly Van Auken

**Comment by epaule** [comment]
That particular gene is from Pristionchus pacificus.
Pristionchus pacificus went though quite a few assembly and gene set updates over the course of time (we did actually skip some, as they were so fast). And that was a gene that was not found anymore when we updated the assembly in 2014.
But the old CDS and protein sequences are still in the respective files as historical sequences (plus the WS2220 release is also in the archive), if you want to do some investigating.
